### ISSUE TITLE
There is no way to exclude peerDependencies when using package-lock.json

### ISSUE THREAD

**Issue by AumyF** [svanderburg/node2nix#253]
Without the option `--lock`, `node2nix` ignores `peerDependencies` by default and we can include them by setting [`--include-peer-dependencies`](https://github.com/svanderburg/node2nix#deploying-peer-dependencies) explicitly.

However, when using `--lock`, `node2nix` includes `peerDependencies` by default and there is no way to make it ignore them.

**Comment by svanderburg** [comment]
Yes, I've explained the rationale in the pull request -- it's because the option is used by node2nix's implementation of the dependency resolution algorithm. Maybe the documentation can be adjusted to make this more clear.
